LayoutTests/imported/w3c/web-platform-tests/close-watcher/user-activation/nnn-CloseWatcher-dialog-popover-expected.txt

```diff
@@ -1,4 +1,4 @@
 button
 
-FAIL Create a CloseWatcher without user activation; create a dialog without user activation; create a popover without user activation promise_test: Unhandled rejection with value: object "ReferenceError: Can't find variable: CloseWatcher"
+FAIL Create a CloseWatcher without user activation; create a dialog without user activation; create a popover without user activation assert_array_equals: lengths differ, expected array ["dialog cancel[cancelable=false]", "CloseWatcher cancel[cancelable=false]", "CloseWatcher close", "dialog close"] length 4, got ["CloseWatcher cancel[cancelable=false]", "CloseWatcher close", "dialog cancel[cancelable=true]"] length 3
```

Comment:
I suppose this failure is still expected.

Reply:
Yeah follow up patches are needed to fully integrate with the popover and dialog elements. Once that's done these tests should all pass correctly.